replies.go

```diff
@@ -180,6 +180,6 @@ var replies = []Reply{
 	},
 	{
 		pattern: `raycon`,
-		message: "Before we start this video, I’d like to thank the sponsor of this video, “Raycons.” I actually have a pair of Raycons and I love them. I usually use them when I’m at the gym or I’m at home playing Minecraft. The sound quality is just as amazing as all the other top name brands and they’re half the price. The ones I’m using are the everyday E25’s. They’re the best ones yet. 6 hours of playtime, seamless Bluetooth pairing, more bass, available in multiple colors, and their compact design helps get rid of background noise. I also like the fact that you can click either earbud with your finger to pause your music. Makes it super convenient if you need to stop for any reason. Go to buyracon.com/(youtubername) for 15% off your order. The link is in the description.",
+		message: "Before we start this video, I’d like to thank the sponsor of this video, “Raycons”. I actually have a pair of Raycons and I love them. I usually use them when I’m at the gym or I’m at home playing Minecraft. The sound quality is just as amazing as all the other top name brands and they’re half the price. The ones I’m using are the everyday E25’s. They’re the best ones yet. 6 hours of playtime, seamless Bluetooth pairing, more bass, available in multiple colors, and their compact design helps get rid of background noise. I also like the fact that you can click either earbud with your finger to pause your music. Makes it super convenient if you need to stop for any reason. Go to buyracon.com/ImpactClient for 15% off your order.",
```

Comment:
mfw buyracon.com is a troll/spam/clickbait meme - their actual website is rayconglobal.com

If this is intentional, at least tell me we know this website doesn't include or link to places that may include dodgy shit and/or viwuses